README.md

```diff
@@ -1,5 +1,7 @@
 # ApiSurface
 
+<img src="logos/text-white.png" alt="ApiSurface logo: a minimalistic cube, dimetric projection, composed of three stacked layers which are joined at some edges so as to suggest the letter S. The visible parts of the stacked layers inside the cube are light blue; the outer surface is white. To the right is the text 'ApiSurface' in the same colour as the cube's exterior." style="height:200px;object-fit:cover"/>
```

Comment:
I think that the logo would look better if it were cropped to its actual dimensions; in the preview that GitHub shows me there's a lot of dead space.

Reply:
You indeed want to try the `picture` tag here, the preview shows

![image](https://github.com/G-Research/ApiSurface/assets/2621499/7401937c-857c-48e8-a620-3b6cfb26f146)


for me.